package.json

```diff
@@ -6,13 +6,14 @@
     "preinstall": "if [ \"$CI\" = \"true\" ]; then echo \"Skipping preinstall...\"; else npx -y only-allow pnpm; fi",
     "install:csb": "pnpm install --frozen-lockfile",
     "test": "pnpm run test:ci",
-    "test:ci": "pnpm run test:format && pnpm run test:eslint && pnpm run test:jest --collectCoverage false && pnpm run typecheck",
+    "test:ci": "pnpm run test:format && pnpm run test:eslint && pnpm run test:jest --collectCoverage false && pnpm run test:vitest && pnpm run typecheck",
     "test:react:17": "pnpm --filter \"./packages/react-*\" --parallel --no-bail run test:jest --collectCoverage false",
     "test:eslint": "pnpm --filter \"./packages/**\" --parallel --no-bail run test:eslint",
     "test:format": "pnpm run prettier --check",
     "test:jest": "pnpm --filter \"./packages/**\" --parallel --no-bail run test:jest",
     "test:size": "pnpm run build && bundlewatch",
-    "build": "rollup --config rollup.config.js && pnpm run typecheck && pnpm pnpm --filter \"eslint-plugin-query\" run build && pnpm run build:copyTypes",
+    "test:vitest": "pnpm --filter \"@tanstack/svelte-query\" run test:vitest",
```

Comment:
I think it would be better to have one test task that every package should implement, otherwise, they won't run in parallel. Naming the task `test:jest` was probably a bad idea, so how about we rename `test:jest` to `test:lib` and then have every package specify that task and implement it with either `jest` or `vitest` ?

Reply:
Yep that makes sense - for now I've just renamed it to "test:jest" to avoid changing the other packages as little as possible in this PR

Reply:
perfect, I can do the rename separately